### ISSUE TITLE
Functional JavaScript

### ISSUE THREAD

**Issue by kimjoar** [superherojs/superherojs#33]
http://blog.fogus.me/2013/03/20/fun-js/

**Comment by kimjoar** [comment]
We have already linked to JavaScript Allongé, which is free to read online. If we want to add this book, we should remove that one. I've read both, but I basically read them both in one go, so I don't remember all the details. Being free to read online counts in favor for Allongé in my mind.

@mollerse @mikaelbr Read both? Which do you prefer?

**Comment by mikaelbr** [comment]
Yeah, both are worth reading I think. I pretty much like all things Reginald Braithwaite, and I think you have a good point with the book now being available online for free. Better suited for a online resource site, maybe?

Allongé is maybe more recipe oriented but still explains the concepts pretty well. It's been a while since I read them, and like you, I read them within a pretty short interval, but I remember thinking "Functional Programming with Underscore.js" being a more lightly read, but liking Allongé better - all though some of the examples and concept might be hard to follow at first.

**Comment by kimjoar** [comment]
Great point, @mikaelbr. Thanks for the input! I'll close this one and keep Allongé as the recommendation for now.

**Comment by mollerse** [comment]
Also pretty much read them back-to-back, but in the other order (Functional.js first).

Functional.js struck me as a more complete and practical introduction. Allongé has a more theoretical approach and focuses on building the understanding up from the core concepts. I think Functional.js is better suited for those who want to learn functional programming in a way that is more directly applicable to "everyday programming", whereas Allongé is better for those who already have an interest in functional programming and want the theoretical foundation.

**Comment by mikaelbr** [comment]
I agree with that Fun.js has much more of a practical approach, but still I really think that Allongé being available online speaks for it - and it being continuously updated. Just yesterday a new big release was shipped. But again, it might alienate someone with it's more advance concept and being too high entry.
